src/components/study/rights/StudyContributorsTable.tsx

```diff
@@ -4,23 +4,19 @@ import Block from '@/components/base/Block'
 import HelpIcon from '@/components/base/HelpIcon'
 import Modal from '@/components/modals/Modal'
 import { FullStudy } from '@/db/study'
-import { isAdminOnStudyOrga } from '@/services/permissions/study'
 import { Post, subPostsByPost } from '@/services/posts'
-import { StudyRole } from '@prisma/client'
 import { ColumnDef, flexRender, getCoreRowModel, useReactTable } from '@tanstack/react-table'
-import { User } from 'next-auth'
 import { useTranslations } from 'next-intl'
 import Link from 'next/link'
 import { useMemo, useState } from 'react'
 
 interface Props {
   study: FullStudy
-  user: User
-  userRoleOnStudy?: StudyRole
+  disabled: boolean
```

Comment:
```suggestion
  disabled?: boolean
```

Comme ça, pas besoin de mettre disabled = false

Reply:
Oui je me disais que ca éviter de forcer tout le temps à mettre disabled mais enfait ici on voudra probablement à chaque fois désactiver pour les mêmes personnes dont ca me va

Reply:
Ha pardon j'avais pas compris ton retour. J'avoue qu'entre les deux j'ai pas de pref. Parcontre enfait je pense que je vais le rendre obligatoire du coup ^^

Reply:
Dans ce cas, je renommerais en "canAddContributor"

Reply:
Je comprends pas pourquoi ca change quelque chose ^^